### PR TITLE
[RF] AT_12.07_001 | Freestyle project>Edit description> Verify Type text

### PR DESCRIPTION
**Pull request by DimaCurkanu**
Correcting a test to change the landing page to a "freestyle page"
Changes made:
redirected to the "freestyle" project page in test #001
added fixtures in every test
code was moved to another file to avoid duplicates
changed the name of the final file to comply with the project rules
https://trello.com/c/N4ppVLwi/946-rf-freestyle-projectedit-description-verify-type-text

### CODE REVIEW

**Review by OLgaByR — COMMENTED** [review]
You have deleted file cypress/e2e/freestyleProjectAddDescription.cy.js , but it had test in it,  but i see that you are the author so i assume it is supposed to be like that?

File: cypress/e2e/freestyleProjectEditDescription.cy.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    cy.get('.hudson_model_FreeStyleProject').click();
+    cy.get('#ok-button').click();
+    cy.get('#general').should('have.text', createNewView.header);
+    cy.get('.jenkins-button--primary').click();
+  })
+  it('AT_12.07_001 | Freestyle project> Edit description> Verify Type text', function () {
+    cy.get('.leading-icon').realHover();
+    cy.get('#description-link').should('be.visible').and('includes.text', createNewView.description).click();
+    cy.get('.jenkins-input').type(createNewView.typeText);
+    cy.get('.jenkins-input').should('be.visible');
```
Comment (OLgaByR) [inline cypress/e2e/freestyleProjectEditDescription.cy.js:20]: why do we need assert about input visibility here?

File: cypress/e2e/freestyleProjectEditDescription.cy.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    cy.get('.jenkins-button--primary').click();
+    cy.get('#description').contains(createNewView.typeText)
+  });
```
Comment (OLgaByR) [inline cypress/e2e/freestyleProjectEditDescription.cy.js:23]: there is final no assert in test , for example assert that typeText is visible could be added (depends on your test case goal)

File: cypress/e2e/freestyleProjectEditDescription.cy.js (hunk 1 above)
Comment (DimaCurkanu) [reply cypress/e2e/freestyleProjectEditDescription.cy.js]: We have two similar files in our project. So I was advised by the reviewers to move my code to another file (there are more tests in it) and change the name of this file according to the rules

**Review by Rom-T — APPROVED** [review]

**Review by S-Lanchik — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Pull request merged**